tests/utils/test_retriever.py

```diff
@@ -47,6 +48,24 @@ def mock_forward(*args: Any, **kwargs: Any) -> Dict[str, torch.Tensor]:
         assert isinstance(result[0], torch.Tensor)
         assert result[0].shape == (1, 3)
 
+        # Test distributive property
```

Comment:
This part aims to check the correctness of some equivalent operations of getting embeddings for lists.

Reply:
cool